### PR TITLE
Fix links 2015 04 26

### PR DESCRIPTION
**Pull request by AchoArnold**
Hey @arschmitz, Please ignore the following links
1.  https://help.github.com/articles/using-pull-requests/
2.  https://help.github.com/articles/github-flavored-markdown/

### CODE REVIEW

**Comment by arschmitz** [comment]
@najela why should we ignore those? They seem like perfectly valid url's there are no redirects or anything? It also looks like you fixed them in this PR?

**Comment by najela** [comment]
image
@arschmitz Looking at **Link 2** and **Link 3** above, you'll see that the source links are those 2 links I posted earlier.

**Comment by scottgonzalez** [comment]
Looks like the links need to be fixed to use HTTPS instead of HTTP.

**Comment by scottgonzalez** [comment]
Oh, no, that's not it. I'm not sure why the output shows the URL with HTTP. Anyway, the difference is the trailing slash.

**Comment by scottgonzalez** [comment]
I figured out where the HTTP comes from. The non-slashed HTTPS URL is getting a 301 to the slashed HTTP URL, which is getting a 307 to the slashed HTTPS URL.

**Comment by najela** [comment]
@scottgonzalez I've updated the PR as per your suggestions.

**Comment by scottgonzalez** [comment]
Can you please update this so it will merge cleanly?

**Comment by najela** [comment]
@scottgonzalez All good? Sorry for the wait :smirk:

**Comment by scottgonzalez** [comment]
The changes look good to me, though it's odd that there's a paragraph rewrite mixed in.

File: page/ajax/ajax-events.md [hunk 1]
```diff
@@ -5,7 +5,8 @@
 	"attribution": [ "jQuery Fundamentals" ]
 }</script>
 
-Often, you'll want to perform an operation whenever an Ajax request starts or stops, such as showing or hiding a loading indicator. Rather than defining this behavior inside every Ajax request, you can bind Ajax events to elements just like you'd bind other events. For a complete list of Ajax events, visit [Ajax Events documentation on docs.jquery.com](http://docs.jquery.com/Ajax_Events).
+
```
Comment (arthurvr) [inline page/ajax/ajax-events.md]: Can you remove the addition of a new blank line in here? They're unrelated changes and there being 2 lines would be inconsistent.

**Comment by arthurvr** [comment]
The links all look good to me but I'd prefer the rewrite of a paragraph not to go inside of a commit to fix links. I like the changes so maybe @najela can extract the changes to that paragraph to its own commit? That would be awesome :)

**Comment by arthurvr** [comment]
Friendly ping @najela.

**Comment by najela** [comment]
Things have been crazy for me this past month but I'll fix this by sunday.

**Comment by arthurvr** [comment]
Thank you @najela.

**Comment by najela** [comment]
I've updated this

**Comment by najela** [comment]
Any Updates?

**Comment by AurelioDeRosa** [comment]
I'm going to take a look at this PR tomorrow. Thank you for your patient and for the PR.

**Comment by AchoArnold** [comment]
Any updates @AurelioDeRosa
